File: src/main/java/org/snust/사탕14568.java
package org.snust;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class 사탕14568 {
  public static void main(String[] args) throws IOException {
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    int candy = Integer.parseInt(br.readLine());
    int answer = 0;
    for (int a = 1; a < candy + 1; a++) {
      for (int b = 1; b < candy + 1; b++) {
        for (int c = 1; c < candy + 1; c++) {
          if (a+b+c == candy){
            if (c >= b+2){
              if (a % 2 == 0){
                answer++;
              }
            }
          }
        }
      }
    }
    System.out.println(answer);
  }
}

File: src/main/python/사탕14568.py
candy = int(input())

answer = 0

# 사탕을 줄 모든 경우의 수
for a in range(1, candy + 1):  # 택희
  for y in range(1, candy + 1):  # 영휸
    for n in range(1, candy + 1):  # 남규
      if a + y + n == candy:
        if y + 2 <= n:
          if a % 2 == 0:
            answer += 1
print(answer)
